app/models/enumeration.rb

```diff
@@ -40,9 +40,9 @@ class Enumeration < ActiveRecord::Base
   before_destroy :check_integrity
 
   validates_presence_of :name
-  validates_uniqueness_of :name, scope: %i(type project_id)
-  validates_length_of :name, maximum: 30
```

Comment:
If this is an intended change, we need to also change the type in the database.

Reply:
No this was intended, thanks for catching